Gemfile

```diff
@@ -24,9 +19,23 @@ gem 'wordnet'
 #gem 'wordnet-defaultdb'
 gem 'sentimental'
 
+# Document uploading and parsing
+gem 'ruby-rtf', '0.0.1'
+gem 'docx-html', '~> 0.1.0'
+gem 'pdf-reader', '1.3.3'
+gem 'nokogiri', '~>1.6.7'
+
 # Development
 group :development, :test do
   gem 'pry'
   gem 'web-console', '~> 2.0'
   gem 'spring'
+  gem 'sqlite3'
+  gem 'byebug', '3.4.0'
+  #gem 'pg
+end
+
+# Production
+group :production do
```

Comment:
Oh, my bad. I thought you switched to `sqlite3` globally. Good use of groups here, no need to flip back to postgres. :)